test/fixtures/rules/std_rule_cases/L016.yml

```diff
@@ -129,3 +129,30 @@ test_compute_line_length_before_template_expansion_2:
         context:
           bi_ecommerce_orders_bi_ecommerce_orders: bq-business-intelligence.user.ecommerce_orders
           table_alias_table_alias_table_alias_table_alias_table_alias_table_alias: t
+
+
+test_long_jina_comment:
+  fail_str: |
+    SELECT *
+    {# comment #}
+    {# ........................................................................... #}
+    FROM table
+  configs:
+    rules:
+      max_line_length: 80
+      L016:
+        ignore_comment_lines: False
+
+
+test_long_jina_comment_ignore:
```

Comment:
IIUC, are we essentially testing for the presence of a bug here (i.e. still failing on Jinja comments)? Should we consider omitting this test?

Reply:
Kind of. However at the moment we are assuming `placeholders` are not treated as comments. And in fact there can be problems if it is assumed to be (the `{% endfor %}` example above. So I still think best to have this test for now.

Plus if we ever add the logic to treat Jinja comments the same as SQL comments, then we'd need a test case and here we have a perfect one already created - just need to update `fail_str` to `pass_str`.